example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -25,8 +24,7 @@ protected boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNSvgPackage()
```

Comment:
I've only been playing with iOS. I'll have to setup an Android environment and test it.